src/test/java/com/tourranger/item/ItemServiceTest.java

```diff
@@ -0,0 +1,108 @@
+package com.tourranger.item;
+
+import com.tourranger.airline.entity.Airline;
+import com.tourranger.airline.repository.AirlineRepository;
+import com.tourranger.item.dto.ItemResponseDto;
+import com.tourranger.item.entity.Item;
+import com.tourranger.item.repository.ItemRepository;
+import com.tourranger.item.service.ItemServiceImpl;
+import com.tourranger.thumbnailImage.entity.ThumbnailImage;
+import com.tourranger.thumbnailImage.repository.ThumbnailImageRepository;
+import com.tourranger.travelAgency.entity.TravelAgency;
+import com.tourranger.travelAgency.repository.TravelAgencyRepository;
+import org.junit.jupiter.api.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
```

Comment:
```suggestion
@TestInstance(TestInstance.Lifecycle.PER_CLASS)
@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
```
TestInstance의 Lifecycle을 클래스 단위로 생성되도록 설정! 하나 배워 갑니다.